pkgs/development/python-modules/jc/default.nix

```diff
@@ -22,6 +25,14 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ ruamel-yaml xmltodict pygments ];
 
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = let emulator = stdenv.hostPlatform.emulator buildPackages; in ''
```

Comment:
What's the emulator doing here?

Reply:
This is to support cross-compilation. I have found this pattern, for example, in `hugo` https://github.com/NixOS/nixpkgs/blob/nixos-23.05/pkgs/applications/misc/hugo/default.nix#L28 .

Reply:
That's an ..interesting pattern. I don't see a quick other way around that problem other than to disable completions on cross so I guess this is how it is.